lints/lint_sub_ca_eku_valid_fields.go

```diff
@@ -0,0 +1,41 @@
+package lints
+
+import (
+	"github.com/zmap/zcrypto/x509"
+	"github.com/zmap/zlint/util"
+)
+
+type subCAEKUValidFields struct {
+	// Internal data here
+}
+
+func (l *subCAEKUValidFields) Initialize() error {
+	return nil
+}
+
+func (l *subCAEKUValidFields) CheckApplies(c *x509.Certificate) bool {
+	return util.IsSubCA(c) && util.IsExtInCert(c, util.EkuSynOid)
+}
+
+func (l *subCAEKUValidFields) RunTest(c *x509.Certificate) (ResultStruct, error) {
+	for _, ekuValue := range c.ExtKeyUsage {
+		if ekuValue == x509.ExtKeyUsageServerAuth ||
+			ekuValue == x509.ExtKeyUsageClientAuth {
+			continue
+		} else {
+			return ResultStruct{Result: Notice}, nil
```

Comment:
I agree that it seems wrong to notice if other values are present. What if the notice was in the situation that neither of those was present rather than if something else is _also_ present?